backend/src/ai/_ollama.py

```
@@ -0,0 +1,79 @@
+from fastapi import APIRouter, Request, HTTPException
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import

(F401)